backend/src/game/player.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.game import constants
+from backend.src.game.locations import Locations
+
+
+class Player:
+    def __init__(self):
+        self.maximum_health = constants.DEFAULT_HEALTH
+        self.current_health = self.maximum_health
+        self.location = Locations.OUTSIDE
+        self.is_alive = bool(0)
+        self.victory_points = constants.ZERO
+
+    def move_to_tokyo(self):
+        self.location = Locations.TOKYO
+
+    def leave_tokyo(self):
+        self.location = Locations.OUTSIDE
+
+    def change_health(self, change_integer):
+        self.current_health += change_integer
+        if self.current_health > self.maximum_health:
+            self.current_health = self.maximum_health
+        if self.current_health <= constants.ZERO:
+            self.is_alive = bool(1)
```

Comment:
In this statement, it should be bool(0).  Also for clarity and just good pratice, use the built-in False.

Reply:
![image](https://user-images.githubusercontent.com/9423528/66616454-ce4ca380-eb9e-11e9-9c8d-c9dbf76fbc58.png)
What do?

Reply:
Right idea but your letting your test go against what would be code readability.  What you should do is make the value use the python built-ins but change the test to say.

` assert not player.is_alive`

The important thing I always remind people with PEP-8 is that it is a style "guide" not a directive. 

https://stackoverflow.com/a/14734103
https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html

Reply:
Coming from Java-lalaland. assert not is what I was looking for.
PEP8 wouldn't be a very good guide if there were situations where you HAD to go against it. I knew there'd be a solution.